NEWS.md

```diff
@@ -19,9 +19,11 @@ Note that each entry is kept to a minimum, see links for details.
   `**nil` is treated similarly to `**{}`, passing no keywords,
   and not calling any conversion methods.  [[Bug #20064]]
 
-* Block passing is no longer allowed in index.  [[Bug #19918]]
+* Block passing is no longer allowed in index assignment 
```

Comment:
A trailing space is intended?

```suggestion
* Block passing is no longer allowed in index assignment
```

Reply:
Not intended.  I pushed a patch to remove that.